### PR TITLE
fix: sec issue about conversion alerts

### PR DESCRIPTION
**Pull request by ruojunm**
### Description

fix: sec issue about conversion alerts

### Rationale

https://github.com/bnb-chain/greenfield-storage-provider/security/code-scanning/8
https://github.com/bnb-chain/greenfield-storage-provider/security/code-scanning/7
https://github.com/bnb-chain/greenfield-storage-provider/security/code-scanning/2

### CODE REVIEW

File: modular/gater/auth_handler.go [hunk 1]
```diff
@@ -5,10 +5,11 @@ import (
 	"encoding/xml"
 	"net/http"
 	"regexp"
-	"strconv"
 	"strings"
 	"time"
 
+	"github.com/bnb-chain/greenfield-storage-provider/util"
```
Comment (BarryTong65) [inline modular/gater/auth_handler.go:11]: the imports should be formatted by stdlib/third party/current_project

File: store/piecestore/storage/checksum.go [hunk 2]
```diff
@@ -56,7 +56,9 @@ func verifyChecksum(rc io.ReadCloser, checksum string) io.ReadCloser {
 	if checksum == "" {
 		return rc
 	}
-	expected, err := strconv.Atoi(checksum)
+	//expected, err := strconv.Atoi(checksum)
```
Comment (BarryTong65) [inline store/piecestore/storage/checksum.go:59]: pls remove the redundant comment

File: store/piecestore/storage/checksum_test.go [hunk 3]
```diff
@@ -80,6 +80,7 @@ func Test_checksumReaderRead(t *testing.T) {
 }
 
 func Test_verifyChecksum(t *testing.T) {
+	//int32Upper := 4294967296
```
Comment (BarryTong65) [inline store/piecestore/storage/checksum_test.go:83]: ditto

**Review by BarryTong65 — APPROVED** [review]
LGTM

File: modular/gater/auth_handler.go (hunk 1 above)
Comment (clydemeng) [reply modular/gater/auth_handler.go]: Done

**Review by annielz — APPROVED** [review]

**Pull request merged**